contracts/oracle/PCVOracle.sol

```diff
@@ -166,32 +148,34 @@ contract PCVOracle is IPCVOracle, CoreRefV2 {
     /// only callable by a liquid pcv deposit that has previously been listed
     /// in the PCV Oracle, because an oracle has to be set for the msg.sender.
     /// this allows for lazy evaluation of the TWAPCV
-    /// @param pcvDelta the amount of PCV change in the venue
+    /// @param deltaBalance the amount of PCV change in the venue
+    /// @param deltaProfit the amount of profit in the venue
     function updateLiquidBalance(
-        int256 pcvDelta
+        int256 deltaBalance,
+        int256 deltaProfit
     )
         public
         onlyVoltRole(VoltRoles.LIQUID_PCV_DEPOSIT)
         isGlobalReentrancyLocked
     {
-        _updateBalance(_getUsdPcvDelta(msg.sender, pcvDelta), true);
-        _afterActionHook();
+        _updateBalance(msg.sender, deltaBalance, deltaProfit, true);
     }
 
     /// @notice update the cumulative and last updated times
     /// only callable by an illiquid pcv deposit that has previously been listed
     /// in the PCV Oracle, because an oracle has to be set for the msg.sender.
     /// this allows for lazy evaluation of the TWAPCV
-    /// @param pcvDelta the amount of PCV change in the venue
+    /// @param deltaProfit the amount of profit in the venue
+    /// @param deltaBalance the amount of PCV change in the venue
     function updateIlliquidBalance(
-        int256 pcvDelta
+        int256 deltaBalance,
```

Comment:
do we really need to pass `deltaBalance` into both updateLiquid and updateIlliquid balance? it feels like the only place this is used is in emitting the event in the `_updateBalance` function.

Reply:
I feel that PCVDeposits emitting their delta Balance + delta Profit to a central entity (PCV Oracle) is the way to go, but I'll remove all storage writes in the PCVOracle for now, and only keep the events emit, to save on gas. We can upgrade the PCVOracle later to an implementation that keeps what we need in storage later.

Reply:
Agreed with having a central place where delta balance + profit are pushed. Will make off-chain accounting far, far easier